examples/todomvc/project.clj

```diff
@@ -1,8 +1,9 @@
 (defproject todomvc-re-frame "0.8.0"
-  :dependencies [[org.clojure/clojure        "1.8.0"]
+  :dependencies [[org.clojure/clojure        "1.9.0-alpha10"]
                  [org.clojure/clojurescript  "1.9.89"]
                  [reagent "0.6.0-rc"]
-                 [re-frame "0.8.0-alpha4"]
+                 [re-frame "0.8.0-SNAPSHOT"]
+                 [day8.re-frame/test "0.1.0-SNAPSHOT"]
```

Comment:
Yep it's up there, 0.1.3 is the latest version: https://clojars.org/day8.re-frame/test